core/tests/webdriverio_utils/DiagnosticTestPage.js

```diff
@@ -97,21 +115,63 @@ var DiagnosticTestPage = function () {
       addTopicToClassroomButton
     );
 
-    await action.setValue(
-      'Add topic ID to classroom input',
-      addTopicToClassroomInput,
-      topicId
+    await AutocompleteTopicDropdownEditor(topicName);
+    await action.waitForAutosave();
+
+    await action.click(
+      'Save classroom config button',
+      saveClassroomConfigButton
     );
+  };
+
+  this.updateClassroomData = async function () {
+    await action.click('Classroom config tile selector', classroomTileSelector);
 
     await action.click(
-      'Add topic ID to classroom submit button',
-      submitTopicIdToClassroomButton
+      'Edit classroom config button',
+      editClassroomConfigButton
+    );
+
+    await action.setValue(
+      'New classroom teaser text input',
+      updateClassroomTeaserTextInput,
+      'teaser text'
+    );
+
+    await action.setValue(
+      'New classroom course details input',
+      updateClassroomCourseDetailsInput,
+      'course details'
     );
 
+    await action.setValue(
+      'New classroom topic list intro input',
+      updateClassroomTopicListIntroInput,
+      'topic list intro'
+    );
+
+    await workflow.submitImage(
+      topicThumbnailButton,
+      thumbnailContainer,
+      '../data/test_svg.svg',
+      false
+    );
+
+    await workflow.submitImage(
+      bannerClickable,
+      bannerCropper,
+      '../data/img.png',
+      false
+    );
```

Comment:
Do we ever assert that these values are set properly? To test that this functionality also works, it'd be good to use `expect(...)` statements to ensure that these values appear correctly to the user, unless that's already tested elsewhere

Reply:
Done.